### ISSUE TITLE
aria test: aria-disabled

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#19]
This repo might be a good place to document the current behavior of aria attributes and roles.  This issue is the first.

Testing the behavior of ```aria-disabled```, which behaves like:

>Indicates that the element is perceivable but disabled, so it is not editable or otherwise operable. See related aria-hidden and aria-readonly.

>For example, irrelevant options in a radio group may be disabled. Disabled elements might not receive focus from the tab order. For some disabled elements, applications might choose not to support navigation to descendants. In addition to setting the aria-disabled attribute, authors SHOULD change the appearance (grayed out, etc.) to indicate that the item has been disabled.

>The state of being disabled applies to the current element and all focusable descendant elements of the element on which the aria-disabled attribute is applied.

(https://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled)

**Comment by jessegreenberg** [comment]
Using this JSFiddle to test: https://jsfiddle.net/7obw983y/6/

- JAWS:
  - Chrome: Attribute works.
  - IE 11: aria-disabled does not apply to all focusable descendant elements. It only works for elements that have the attribute directly.
  - Firefox: Attribute works.

- NVDA:
  - Chrome: Attribute works.
  - IE 11: aria-disabled does not apply to all focusable descendant elements. It only works for elements that have the attribute directly.
  - Firefox: Attribute works.

- VoiceOver:
  - Chrome: Attribute works.  For a generic div, the information comes later in help text as "This item is dimmed".
  - Safari: Attribute works.  For a generic div, the information comes later in help text as "This item is dimmed".
  - Firefox: Attribute works.

**Comment by jessegreenberg** [comment]
@terracoda @emily-phet in this case, IE11 does not support one aspect of this attribute.  As we continue to test these things, should we start submitting bug reports to browsers for missing ARIA 1.0 support?

**Comment by terracoda** [comment]
@jessegreenberg, I think this repo is a great idea, and that submitting bug reports to browsers is also a good idea. It is a way to create our documentation and a  way to make positive change to ARIA support!

**Comment by emily-phet** [comment]
@jessegreenberg @terracoda 
Agreed!

**Comment by emily-phet** [comment]
Since we’ve moved away from IE11, if the bug report has been submitted I think this can be closed. @jessegreenberg

**Comment by terracoda** [comment]
@emily-phet, maybe adding labels is better than closing in case we get a report back.

**Comment by jessegreenberg** [comment]
A bug report has actually not been submitted for this yet. Since we are no longer testing/supporting JAWS+IE11 it seems like this issue can be closed but @emily-phet @terracoda let me know if I should submit a report.

**Comment by terracoda** [comment]
If we are no longer testing/supporting JAWS+IE11, then I don't think we need to do a bug report. @emily-phet, do you agree?

**Comment by emily-phet** [comment]
Agreed. Closing
